Sources/PointFree/MinimalNavView.swift

```diff
@@ -11,6 +11,7 @@ import PointFreeRouter
 import Styleguide
 import Prelude
 import View
+import Views
```

Comment:
this import is now needed because I moved our SVG's to the views framework. I'll comment more about this in a moment.